content/downloads/mac.html

```diff
@@ -38,7 +38,7 @@ <h3>Binary installer</h3>
         let ago = "" + daysAgo + " days ago"
 
         const handwave = (exact, unit) => {
-          const rest = (count) => "" + count + " " + unit + (unit > 1 ? "s" : "")
+          const rest = (count) => "" + count + " " + unit + (count > 1 ? "s" : "") + " ago"
```

Comment:
I guess it'd be even better to use this form:

```suggestion
          const rest = count => `${count} ${unit}${count > 1 ? "s" : ""} ago`
```